src/main/java/seedu/address/model/participation/Participation.java

```diff
@@ -34,6 +35,10 @@ public Person getStudent() {
     public Tutorial getTutorial() {
         return tutorial;
     }
+    public static Boolean isValidParticipationList(String test) {
+        return test.matches(VALIDATION_REGEX);
```

Comment:
So the VALIDATION_REGEX used here is the same one as Address? Maybe checking for participation is not the most necessary here 😅